intlogger.go

```diff
@@ -65,6 +77,12 @@ func New(opts *LoggerOptions) Logger {
 		output = DefaultOutput
 	}
 
+	if fi, ok := output.(*os.File); ok && opts.Color {
+		output = colorable.NewColorable(fi)
+	} else if opts.Color {
+		panic("Cannot enable coloring of non-file Writers")
+	}
```

Comment:
So looking at colorable, the restriction here for os.File is only true on windows. Given this code will run on... less restrictive platforms, I think it might make sense to break this out and for any unix platform allow the output to be used directly and only run it through `colorable` on non-unix platforms.

Reply:
Ok, I tackled this using build tags. The other option would be to check `runtime.GOOS` at runtime, which is less mental overhead. I pushed this change.

Reply:
Build tags are the right way, nice.